### DISCUSSION TITLE
Reset password form asks users to sign in even for roles without app access

### DISCUSSION THREAD

**Discussion by paulleduc** [directus/directus#24945] [Feature Requests]
### Describe the Bug

I am hosted on Directus Cloud, but I've build a custom UI for all non-admin users. I've disabled app access for these users because they should never be allowed to access the admin dashboard. With Directus Cloud, I'm not able to send password reset emails with a custom reset_url, so I have to use the built in one. 

The password reset flow is confusing for my regular users, not only because of the change of UI/domain, but also because the reset password form asks the users to sign in to the admin dashboard upon resetting the password. Attempting to log in fails, but it's just a confusing flow when the role is known to not have app access.

### To Reproduce

Send a password reset email to a user with a role that has app access disabled. Reset the password.

### Errors Shown

N/A

### What version of Directus are you using?

9.16.1

### What version of Node.js are you using?

16.17.0

### What database are you using?

Directus Cloud

### What browser are you using?

Chrome

### How are you deploying Directus?

Directus Cloud

**Comment by rijkvanzanten** [comment]
Linear: [ENG-194](https://linear.app/directus/issue/ENG-194/reset-password-form-asks-users-to-sign-in-even-for-roles-without-app)
